webrender/src/frame.rs

```diff
@@ -746,11 +460,11 @@ impl Frame {
 
         if level == 0 && self.frame_builder_config.enable_scrollbars {
             let scrollbar_rect = LayerRect::new(LayerPoint::zero(), LayerSize::new(10.0, 70.0));
-            context.builder.add_solid_rectangle(&scrollbar_rect,
-                                                &ClipRegion::simple(&scrollbar_rect),
-                                                &DEFAULT_SCROLLBAR_COLOR,
-                                                PrimitiveFlags::Scrollbar(self.root_scroll_layer_id.unwrap(),
-                                                                          4.0));
+            context.builder.add_solid_rectangle(
```

Comment:
I'm not sure, but the PrimitiveFlags::Scrollbar(self.root_scroll_layer_id.unwrap(), line wouldn't fit within the line length limit any other way.